results/user-testing-results.md

```diff
@@ -0,0 +1,138 @@
+# Results: JupyterLab Accessibility Overview
+
+These results are from user interviews conducted in May 2023 with [this test script]() on [a modified version of NumPy Tutorials’ Saving and sharing your NumPy arrays notebook](https://github.com/Quansight-Labs/JupyterLab-user-testing/blob/main/notebooks/save-load-arrays.ipynb). 
+
+This work was funded by a [Chan Zuckerberg Initiative Essential Open Source Software for Science grant](https://chanzuckerberg.com/eoss/proposals/inclusive-and-accessible-scientific-computing-in-the-jupyter-ecosystem/). The [full grant proposal can be found on the jupyter/accessibility repository](https://github.com/jupyter/accessibility/blob/da6db43da092255c205ce1603d0d10b1f51c131b/docs/funding/Inclusive_and_Accessible_Scientific_Computing_in_Jupyter_Ecosystem_SUBMITTED_PROPOSAL.pdf).
+
+## What was tested
+
+**Operating systems:** iOS, Mac OS Monterey, Windows 10, Windows 11
+
+**Browsers:** Chrome, Firefox, Opera GX, Safari
+
+**Assistive tech:** Built-in large cursor and pointer operating system settings, customized operating system shortcuts, Dragon Professional (speech recognition software), in-browser contrast adjustments, in-browser focus styling adjustments, Mac OS Zoom (built-in screen magnifier), mobile device, split keyboard, [sticky keys](https://en.wikipedia.org/wiki/Sticky_keys)
+
+**Interface:** Participant browser and [preset environment in The Littlest JupyterHub](https://github.com/Quansight-Labs/JupyterLab-user-testing#jupyterlab-user-testing-environment) running JupyterLab 3.6.1. 
+
+**Sample size:** 7 participants
+
+**Method:** Combination of qualitative [usability testing](https://www.nngroup.com/articles/usability-testing-101/) and [user interviews](https://www.nngroup.com/articles/user-interviews/)
+
+**Team:** [Isabela Presedo-Floyd](https://github.com/isabela-pf), [Tania Allard](https://github.com/trallard/), [Stephannie Jimenez Gacha](https://github.com/steff456), and [Gabriel Fouasnon](https://github.com/gabalafou/) via [Quansight Labs](https://labs.quansight.org/)
+
+## Limits
```

Comment:
I'm open to a title change. For context, I used `Limits` because I think it is closer to the terminology used in research settings. I could be wrong about this.

[Wikipedia describes disclaimers](https://en.wikipedia.org/wiki/Disclaimer) as more focused on explicitly stating rights  and obligations, and I'm not sure what obligations we are clarifying here so it was not my default. Totally open if I'm reading too much into the words and this is the more standard term.

Reply:
Maybe limitations is better

Reply:
```suggestion
## Limitations
```